descriptive-link-5effbb.md

```diff
@@ -0,0 +1,205 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links in their context describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Carlos Duarte
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element with the [semantic role](#semantic-role) of [link](https://www.w3.org/TR/wai-aria/#link) or a role that inherits from the link role, that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an [accessible-name](#accessible-name) that does not only consist of [whitespace](#whitespace).
+
+## Expectation
+
+The [visible](#visible) [text content](#text-content) of the target element, together with its [programmatically determined link context](#programmatically-determined-link-context) describe the purpose of the link.
```

Comment:
To the definition that is being worked on #693

Reply:
As part of #447 (and per @WilcoFiers suggestion), I have been (re)working the definitions of [context](https://github.com/act-rules/act-rules.github.io/blob/update-SC2-4-6-descriptive-label/pages/glossary/context.md) using [visual context](https://github.com/act-rules/act-rules.github.io/blob/update-SC2-4-6-descriptive-label/pages/glossary/visual-context.md) and [programatically determined context](https://github.com/act-rules/act-rules.github.io/blob/update-SC2-4-6-descriptive-label/pages/glossary/programatically-determined-context.md) (not restricted to programatically determined **link** context).

Given that this seems to be also needed for this rule, and for #693 (which I've also inherited from @annethyme), I strongly suggest that the two of us get on a call soonish and come up with defs that we both agree on.
(maybe as separate PR since that would not require a Final Call if it's Glossary only, and we don't want 3 big PR do have mutual dependencies preventing us from merging…)